OrbitCore/OrbitFunction.h

```diff
@@ -47,57 +39,36 @@ class Function {
            uint64_t address, uint64_t load_bias, uint64_t size,
            std::string_view file, uint32_t line);
 
-  const std::string& Name() const { return name_; }
-  const std::string& PrettyName() const {
-    return pretty_name_.empty() ? name_ : pretty_name_;
-  }
-  std::string Lower() { return ToLower(PrettyName()); }
-  const std::string& GetLoadedModulePath() const { return loaded_module_path_; }
-  std::string GetLoadedModuleName() const {
-    return Path::GetFileName(loaded_module_path_);
+  const std::string& name() const { return name_; }
+  const std::string& pretty_name() const { return pretty_name_; }
+  const std::string& loaded_module_path() const { return loaded_module_path_; }
+  void set_loaded_module_path(const std::string& loaded_module_path) {
+    loaded_module_path_ = loaded_module_path;
   }
-  void SetModulePathAndAddress(std::string_view module_path,
-                               uint64_t module_adddress) {
-    loaded_module_path_ = module_path;
-    module_base_address_ = module_adddress;
+  uint64_t module_base_address() const { return module_base_address_; }
+  void set_module_base_address(uint64_t module_base_address) {
+    module_base_address_ = module_base_address;
   }
-  uint64_t Size() const { return size_; }
-  const std::string& File() const { return file_; }
-  uint32_t Line() const { return line_; }
-  int CallingConvention() const { return calling_convention_; }
-  const char* GetCallingConventionString();
-  void SetCallingConvention(int calling_convention) {
+  uint64_t size() const { return size_; }
+  const std::string& file() const { return file_; }
+  void set_file(std::string& file) { file_ = file; }
```

Comment:
Did you want to pass a *const*-ref? Either way, pass by value and move.

Reply:
Fixed, thanks!